src/DependencyInjection/SatelliteDependencyInjection.php

```diff
@@ -15,12 +12,18 @@
 
 final class SatelliteDependencyInjection
 {
+    public function __construct(private array $providers = [])
+    {
+    }
+
     public function __invoke(array $config): ContainerBuilder
     {
         $container = new ContainerBuilder();
-        $container->addExpressionLanguageProvider(new Provider());
-        $container->addExpressionLanguageProvider(new StringExpressionLanguageProvider());
-        $container->addExpressionLanguageProvider(new ArrayExpressionLanguageProvider());
+        if (count($this->providers) > 0) {
+            foreach ($this->providers as $provider) {
+                $container->addExpressionLanguageProvider(new $provider());
+            }
+        }
```

Comment:
Le `if` ne te sert à rien

```suggestion
        foreach ($this->providers as $provider) {
            $container->addExpressionLanguageProvider(new $provider());
        }
```